tests/unit/test_events.py

```diff
@@ -0,0 +1,120 @@
+import github3
+from unittest import TestCase
+from .helper import UnitHelper
+from .helper import create_example_data_helper
+
+get_example_data = create_example_data_helper('event_example')
+get_org_example_data = create_example_data_helper('org_example')
+get_comment_example_data = create_example_data_helper('comment_example')
+get_pull_request_example_data = create_example_data_helper('pull_request_example')
+get_issue_example_data = create_example_data_helper('issue_example_data')
+get_user_example_data = create_example_data_helper('user_example')
+get_repo_example_data = create_example_data_helper('repo_example')
+get_gist_example_data = create_example_data_helper('gist_example_data')
+get_team_example_data = create_example_data_helper('team_example')
+
+
+class TestEvent(UnitHelper):
+
+    described_class = github3.events.Event
+    example_data = get_example_data()
+
+    def test_equality(self):
```

Comment:
These equality tests probably need to be placed elsewhere, not on these instances. I have to think about where they belong. (They're relics of the old testing strategy).